### DISCUSSION TITLE
install page keeps loading

### DISCUSSION THREAD

**Discussion by salman0ansari** [langgenius/dify#3526] [Help] [answered]
its been 10min and its just keep loading 
![image](https://github.com/langgenius/dify/assets/30106022/dfd7dcfb-94c8-4f67-b8c4-abf5329b9b28)

**Comment by saminbj** [comment]
+1

**Comment by crazywoola** [comment, marked as answer]
Please set the `.env` according to docs. 

https://docs.dify.ai/getting-started/install-self-hosted/local-source-code#follow-these-steps

`cp .env.example .env`

**Comment by StarterES-sgaleano** [comment]
Hi @salman0ansari im having the same problem and i just edit the env file as requested on the docs but still nothing. You marked this as solved... care to share how?
Thanks in advance.

**Reply by salman0ansari** [reply]
i use linux with 2 drives 
ssd (linux) and hdd

i was running this project in hdd but when i moved the code to ssd it worked magically 
maybe it was something related to permissions idk

**Reply by salman0ansari** [reply]
also recheck your .env file

**Comment by omesh845** [comment]
It doesn't work, I am running via docker-compose.yaml running on my local machine without ssl works fine, i m able to register and everything, when I host in vps its working but only when there is no reverse proxy,which  this compromises my ssl and can't map my custom domain, when I link my reverse proxy with SSL it doesn't work - tested with coolify and also individual(seperate) (caddy,treafik,ngix) it's just struck on registration page loading, when I use localhost it's works fine, please help with, there is no point of deploying on our own server of its not SSL enabled and custom domain.

**Reply by elenacliu** [reply]
same problem
